data/json/monsters/insect_spider.json

```diff
@@ -74,8 +74,8 @@
     "default_faction": "worm",
     "bodytype": "snake",
     "species": [ "WORM" ],
-    "volume": "62500 ml",
-    "weight": "81500 g",
+    "volume": "45 L",
+    "weight": "60 kg",
```

Comment:
But this is still more than half the full worm, no?

Reply:
It is more than half the full _smallest_ worm, yes. That size variant does not split. When it grows it becomes 120kg and 92.5 l of volume and only then can it split.